openedx/core/djangoapps/programs/utils.py

```diff
@@ -266,7 +266,7 @@ def reshape(course_run):
                 # count towards completion of a course in a program). This may change
                 # in the future to make use of the more rigid set of "applicable seat
                 # types" associated with each program type in the catalog.
-                'type': course_run['type'],
+                'type': 'verified' if course_run['type'] == 'credit' else course_run['type'],
```

Comment:
Please leave a note about why this is being done, and the ticket number that tracks cleanup (LEARNER-1274).